tests/unit/store/api.spec.js

```diff
@@ -0,0 +1,635 @@
+        mockerror = false
+      })
+      it('should set the version', async () => {
+        await store.dispatch('getApi')
+        expect(store.getters.version).toBe(2)
+      })
+      it('should set the release timestamp', async () => {
+        await store.dispatch('getApi')
+        expect(store.getters.releaseTimestamp).toBe('timestamp')
+      })
+      it('should set connected state', async () => {
+        await store.dispatch('getApi')
+        expect(store.getters.connected).toBe(true)
+      })
+      it('should set api mode', async () => {
+        await store.dispatch('getApi')
+        expect(store.getters.apiMode).toBe('maintenance')
+      })
+    })
+
+    describe('getApi - with api error', () => {
+      beforeEach(() => {
+        mockerror = true
+        const state = localStoreAPIState()
+        store = new Vuex.Store({
+          state: state,
+          getters: api.getters,
+          mutations: api.mutations,
+          actions: api.actions
+        })
+      })
+      it('should unset the version', async () => {
+        expect(store.getters.version).toBe(8)
+        await store.dispatch('getApi')
+        expect(store.getters.version).toBe(null)
+      })
+      it('should unset the release timestamp', async () => {
+        await store.dispatch('getApi')
+        expect(store.getters.releaseTimestamp).toBe(null)
+      })
+      it('should unset connected state', async () => {
+        await store.dispatch('getApi')
+        expect(store.getters.connected).toBe(false)
+      })
+    })
+
+    describe('monitorConnection - no query error', () => {
+      beforeEach(() => {
+        mockerror = false
+        const state = localStoreAPIState()
+        store = new Vuex.Store({
+          state: state,
+          getters: api.getters,
+          mutations: api.mutations,
+          actions: api.actions
+        })
+      })
+      it('should unset connection timeout', async () => {
+        await store.dispatch('monitorConnection')
+        expect(clearTimeout).toHaveBeenCalled()
+      })
+      it('should set the release timestamp', async () => {
+        await store.dispatch('monitorConnection')
+        expect(store.getters.releaseTimestamp).toBe('timestamp')
+      })
+      it('should set connected state', async () => {
+        await store.dispatch('monitorConnection')
+        expect(store.getters.connected).toBe(true)
+      })
+      it('should set the version', async () => {
+        await store.dispatch('monitorConnection')
+        expect(store.getters.version).toBe(2)
+      })
+      it('should set the connection message', async () => {
+        await store.dispatch('monitorConnection')
+        expect(store.getters.connectionMessage).toBe('Connected')
+      })
+      it('should set retries back to 0', async () => {
+        await store.dispatch('monitorConnection')
+        expect(store.getters.retries).toBe(0)
+      })
+    })
+    //Mock won't work as error policy set to 'none'
```

Comment:
If we can't test this part of the store, should we change the error policy on the `fallbackApolloClient` to let us catch issues? Doesn't necessarily need to be in this PR but seems like something we should be able to test.

Reply:
Actually, I updated the mock and can now test this.  We also added a catch for if no data returned.  Tested locally and can't see any error.  (Not actually sure where we use that connectionMessage?)